.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Can we `.gitignore` this file?

Reply:
do you want this completely excluded or just ignored from here on out?

Reply:
Gif ignore the whole directory since people will have different editor settings

Reply:
done